example-2/evm/test/Hub.t.sol

```diff
@@ -155,8 +171,31 @@ contract HubTest is Test, HubStructs, HubMessages, HubGetters, HubUtilities, Tes
 
         doRegisterFakeSpoke();
 
-        setPrice(assets[0], 100);
-        setPrice(assets[1], 90);
+        if(oracleMode == 1){
+            blockTs += 1;
+
+            // asset 0
+            int64 price0 = 100;
+            uint64 conf0 = 0;
+            int32 expo0 = 0;
+            int64 emaPrice0 = 100;
+            uint64 emaConf0 = 100;
+            uint64 publishTime0 = blockTs;
```

Comment:
looks good to me! I think the only thing that would probably improve the ease of writing tests a lot is if you moved this 'int64 price0 = 100;
            uint64 conf0 = 0;
            int32 expo0 = 0;
            int64 emaPrice0 = 100;
            uint64 emaConf0 = 100;
            uint64 publishTime0 = blockTs;
            hub.setMockPythFeed(assets[0].pythId, price0, conf0, expo0, emaPrice0, emaConf0, publishTime0);' stuff all into the setPrice function in TestHelpers (maybe setPrice could take an extra parameter 'oracleMode', and we hardcode everything but the price value, for the sake of test helpers.) after this its good to merge!